en/SetupView/Power.md

```diff
@@ -1,3 +1,43 @@
 # Power Setup
 
-![](../../images/setup/PX4Power.jpg)
\ No newline at end of file
+The *Power Setup* screen is used to configure battery parameters and also provide advanced settings for propellers.
+
+![Battery Calibration](../../images/setup/PX4Power.jpg)
+
+## Battery Voltage/Current Calibration
+
+Enter data for your battery/power module from its data sheet: number of cells, full voltage per cell, empty voltage per cell. If provided, also enter voltage divider and amps-per-volt information. 
+
+*QGroundControl* can be used to calculate appropriate voltage divider and amps-per-volt values from measurements:
+
+1. Measure the voltage from the battery using a multimeter.
+1. Click **Calibrate** next to the *Voltage divider* field. On the prompt that appears:
```

Comment:
It's Calculate not Calibrate. Same below

Reply:
Thanks. Fixed